### PR TITLE
W-14663399-add-configXML-duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 1]
```diff
@@ -60,6 +60,7 @@ The listener is for triggering the flow, and Set Payload provides some test data
 
 . In the Explorer view, select the XML file for your project, such as `my-project-name.xml`:
 +
+//TODO: UPDATE THIS OUT-OF-DATE IMAGE
```
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
```

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 2]
```diff
@@ -258,28 +256,29 @@ Alternatively, use the menu.
...
-    
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" 
+      xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+  
 </mule>
 --
++
+The file's canvas UI provides the same options for building your project file that  appear when you create an integration project.
```
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
The file's canvas UI provides the same options for building your project file that appear when you create an integration project. For an example, see <<add-components, Add a Component to Your Project>>.
```

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 3]
```diff
@@ -239,50 +239,38 @@ Before publishing, run your application in debug mode.
...
-====
-Select the `mule` folder, and then:
-
-** In the desktop IDE, select *File* > *New File*.
-** In the cloud IDE, click the image:icon-menu.png["",18,18] (menu) icon, and select *File* > *New File*.
-====
-. Enter a name for the new file with the `.xml` extension.
-. In the Editor view for the new file, press Ctrl+Space to display available options.
-. Select *Empty mule configuration*:
+The name must be unique within the project. Permitted special characters are `_` (underscore), `.`, `@`, `(`, and `)`.
```
Comment (jiterman) [inline modules/ROOT/pages/int-create-integrations.adoc]: `-` (hyphen) is also accepted :)

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 4]
```diff
@@ -239,50 +239,38 @@ Before publishing, run your application in debug mode.
...
+Anypoint Code Builder creates the initial Mule configuration XML file in `src/main/mule` within the Mule project.
 
-You can create additional configuration files, for example to store global configurations.
+You can create additional configuration files, for example, for an implementation file for a scaffolded interface or for global configurations that you can reference from an XML file. 
 
-To create a new XML configuration file:
+To create a new configuration XML file:
 
-. In the Explorer view, right-click the *mule* folder (`src/main/mule`) and select *New File*.
+. In the Explorer view, right-click the *mule* folder (`src/main/mule`) and select *New Mule Configuration File...* to launch a configuration field for the new file.
```
Comment (jiterman) [inline modules/ROOT/pages/int-create-integrations.adoc]: This is just my opinion, but the last part seems a bit redundant. I'd stick to:

"In the Explorer view, right-click the *mule* folder (`src/main/mule`) and select *New Mule Configuration File...*"

Especially considering that the config field is mentioned in the following point.

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 3 above)
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
The name must be unique within the project. Permitted special characters are `_` (underscore), `-` (hyphen), `.`, `@`, `(`, and `)`.
```

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 4 above)
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
. In the Explorer view, right-click the *mule* folder (`src/main/mule`) and select *New Mule Configuration File...*.
```

**Review by jiterman — APPROVED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**